### PR TITLE
created User model with devise, set i18n and add css to member pages

### PR DESCRIPTION
**Pull request by oSeanChen**
先補上會員的model User 把連結接上首頁，有需要會員系統的功能可以接著開發。

### CODE REVIEW

File: e iCoder_development [hunk 1]
```diff
@@ -0,0 +1,61 @@
+Available help:
```
Comment (noracami) [inline e iCoder_development:1]: no need

File: test/fixtures/users.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (noracami) [inline test/fixtures/users.yml:1]: we will test with RSpec,
so we can remove all test folders and files.

File: test/models/user_test.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```
Comment (noracami) [inline test/models/user_test.rb:1]: same reason

**Review by Yojijun — DISMISSED** [review]

**Review by chutienyuan — APPROVED** [review]

**Review by zxcvbnm7548 — APPROVED** [review]

**Review by noracami — APPROVED** [review]
LGTM

**Review by RexXKao — APPROVED** [review]
LGTM

**Review by Yojijun — APPROVED** [review]

**Review by chutienyuan — APPROVED** [review]

**Pull request merged**